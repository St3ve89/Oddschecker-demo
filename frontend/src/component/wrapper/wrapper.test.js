import { Wrapper } from '.';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';

test('renders yobota table body component', () => {
  const store = {
    getState: () => ({
      oddschecker: {
        data: [],
      },
    }),
    subscribe: jest.fn(),
    dispatch: jest.fn(),
  };
  const component = renderer
    .create(
      <Provider store={store}>
        <Wrapper />
      </Provider>
    )
    .toJSON();
  expect(component).toMatchSnapshot();
});
