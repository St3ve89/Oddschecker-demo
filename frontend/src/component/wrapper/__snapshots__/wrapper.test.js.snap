// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders yobota table body component 1`] = `
<div
  className="wrapperstyle__Container-mimsy5-0 ivpdkG"
>
  <div
    className="wrapperstyle__Header-mimsy5-1 frqRkq"
  >
    <h1
      className="wrapperstyle__Title-mimsy5-2 ieixJp"
    >
      Betslip
    </h1>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline"
      onClick={[Function]}
    >
      <div
        aria-haspopup="listbox"
        className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        role="button"
        tabIndex={0}
      >
        decimalOddsMoreThanTwo
      </div>
      <input
        aria-hidden={true}
        className="MuiSelect-nativeInput"
        onAnimationStart={[Function]}
        onChange={[Function]}
        tabIndex={-1}
        value="decimalOddsMoreThanTwo"
      />
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSelect-icon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </div>
  <div
    className="wrapperstyle__CardContainer-mimsy5-3 gHvgtP"
  />
  <button
    className="buttonstyle__CustomButton-sc-175iax6-0 cFjAcq"
    onClick={[Function]}
  >
    <span>
      Bet now
    </span>
  </button>
</div>
`;
